torch/csrc/generic/methods/Tensor.cwrap

```diff
@@ -326,20 +326,36 @@ PyObject * THPTensor_(stride)(PyObject *self, PyObject *args, PyObject *kwargs)
     - THBoolTensor* mask
 ]]
 
+THTensor* THTensor_(transpose_neg)(THTensor *self, THTensor *src, int dim0, int dim1)
```

Comment:
You'll need to take in a THCState* for CUDA tensors:

```
#if IS_CUDA
THTensor* THTensor_(transpose_neg)(THCState* state, , THTensor *self, THTensor *src, int dim0, int dim1)
#else
THTensor* THTensor_(transpose_neg)(THTensor *self, THTensor *src, int dim0, int dim1)
#endif
```

Then, in the calls to THTensor_(...)(...) you should pass `LIBRARY_STATE` before self:

```
THTensor_(transpose)(LIBRARY_STATE self, src, dim0, dim1);
```